src/pe_reports/data/db_query.py

```diff
@@ -20,10 +20,9 @@
 
 def show_psycopg2_exception(err):
     """Handle errors for PostgreSQL issues."""
-    err_type, traceback = sys.exc_info()
+    err_type, err_obj, traceback = sys.exc_info()
     line_n = traceback.tb_lineno
-    logging.error(f"\npsycopg2 ERROR: {err} on line number: {line_n}")
-    logging.error(f"psycopg2 traceback: {traceback} -- type: {err_type}")
+    logging.error(f"Database connection error: {err} on line number: {line_n}")
```

Comment:
If you're not using `line_n` anywhere else, I'd just get rid of it and change the logging statement to this:
```suggestion
    logging.error(f"Database connection error: {err} on line number: {traceback.tb_lineno}")
```

Reply:
Fixed [here](https://github.com/cisagov/pe-reports/pull/114/commits/c76a555f77c6e7bc613bcf3dbbdf499361f251de)